### PR TITLE
[FIX] account: fine tuning of https://github.com/odoo/odoo/pull/13463…

### PR DESCRIPTION
**Pull request by fmdl**
…4/commits/056caa296539fdebc39bc07d045aaa21c87ff142

In multi company env use sudo to check.

@yosa-odoo 
@william-andre

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/account/models/partner.py [hunk 1]
```diff
@@ -603,6 +603,6 @@ def _merge_method(self, destination, source):
         """
         Prevent merging partners that are linked to already hashed journal items.
         """
-        if self.env['account.move.line'].search([('move_id.inalterable_hash', '!=', False), ('partner_id', 'in', source.ids)], limit=1):
+        if self.env['account.move.line'].sudo().search([('move_id.inalterable_hash', '!=', False), ('partner_id', 'in', source.ids)], limit=1):
```
Comment (william-andre) [inline addons/account/models/partner.py:606]: We need to also add `active_test=False`

**Review by william-andre — COMMENTED** [review]
Good catch! 

Could you please fix the commit message to mention the commit that is actually merged and not one from the staging?
a40659bcda1a4d635f2caacffb740b001bed9e30

File: addons/account/models/partner.py (hunk 1 above)
Comment (fmdl) [reply addons/account/models/partner.py:606]: why ? the field active is not present on aml ?

Comment (william-andre) [reply addons/account/models/partner.py:606]: Indeed, I have no idea how I came up with this :thinking:

**Review by william-andre — COMMENTED** [review]
@robodoo r+

**Review by william-andre — APPROVED** [review]

**Comment by william-andre** [comment]
@yosa-odoo

**Pull request closed without merging**